client/src/components/NavigationBar.vue

```diff
@@ -42,34 +42,56 @@ export default {
             this.runningJobIds.splice(this.runningJobIds.indexOf(jobId), 1);
           }
           break;
+        default:
```

Comment:
I know this isn't related to changes that you made, but can we insert a comment after
`case jobStatus.SUCCESS.value:`
`// fall through`
Just so we indicate that this is an intentional fall through in the switch statement.
Might be a discussion if we should turn on the linting rule for this.